docs/source/extend_kedro/common_use_cases.md

```diff
@@ -8,7 +8,7 @@ The execution timeline of a Kedro pipeline can be thought of as a sequence of ac
 
 At different points in the lifecycle of these components, you may want to add extra behaviour. For example, you could add extra computation for profiling purposes _before_ and _after_ a node runs or _before_ and _after_ the I/O actions of a dataset, namely the `load` and `save` actions.
 
-This can now achieved by using [Hooks](./hooks.md), to define the extra behaviour and at which point in the execution timeline it should be injected.
+This can now achieved by using [Hooks](../hooks/hooks.md), to define the extra behaviour and at which point in the execution timeline it should be injected.
```

Comment:
```suggestion
This can now achieved by using [Hooks](../hooks/hooks.md), to define the extra behavior, and when in the execution timeline it should be introduced.
```